server/app/router.js

```diff
@@ -56,6 +56,16 @@ router.get(
   authenticateUser,
   transaction.readTransactionById
 );
+router.get(
+  "/transactionsamount/:periodFilter",
```

Comment:
**issue**: What's that endpoint name ? What are you returning to the user, just an amount or a transaction ?

Reply:
What is returned is the total amount of all user's transactions, for a period of time (a day, a week, a month or a year).
It returns only the amount (as a string).

Reply:
Uhh... But does the dashboard display the transactions within that period ?

Reply:
No because the first "part" of the dashboard is supposed to display the sum of all the transactions amounts for a given period and the percentage (and amount) of that total for transactions categories that are part of it.
The goal is to be able in one "click" to know how much of our expenses each category represent.

Reply:
It's unfortunate that you won't have a proper period filter for the presentation, but if it's a first version thing, then why not